examples/app.R

```diff
@@ -5,7 +5,12 @@ VERSION = "1.2.2"
 
 shinyApp(
   ui = tagList(
-    shiny.info::git_info(position = "bottom left"),
+    shiny.info::infoPanel(
```

Comment:
The variable styling is quite a delicate thing in shiny world. We prefer snake_case usually, but for consistency you could introduce `infoPanel` as a wrapper to `info_panel`